stimela/backends/kube/run_kube.py

```diff
@@ -145,10 +145,15 @@ def dprint(level, *args, **kw):
 
                 # apply pod type specifications
                 if kube.dask_cluster.worker_pod:
+                    print(kube.dask_cluster.worker_pod)
```

Comment:
@landmanbester These debug print statements ought to be replaced with dprint(1, content). See examples in same file. That way you can enable the printing by setting ``debug: verbose: 1`` in the backend options.

Reply:
Cool, didn't know that one. Thanks